BEPs/BEP322.md

```diff
@@ -288,6 +288,62 @@ Response:
 }
 ```
 
+##### 4.4.1.3 Query Bundle Data
+
+This API is used by mev data analysts to query the bundle information from the builder. For example, In order to be able
+to directly reflect the mev market size, the mev data provider launches a bundle panel needs to show the number of
+bundles per day or per block.
+
+Request:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "mev_bundles",
+  "params": [
+    {
+      "fromBlock": "0x1",
+      "toBlock": "0x3"
+    }
+  ],
+  "id": 1
+}
+```
+
+Params:
+
+- `fromBlock`: Either the hex value of a block number OR block tags: "latest", "pending" or nil. Defaults to "latest".
+- `toBlock`: Either the hex value of a block number OR block tags: "latest", "pending" or nil. Defaults to "latest". It
+  should be no less than `fromBlock` and less than `fromBlock+100`.
+
+Response:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": [
+    {
+      "fromBlock": "0x1",
```

Comment:
will this introduce too many duplicated data for an indexer when it try to catch up?

Reply:
Added a note to return only bundles whose height is within the block range when the bundle was first received, regardless of the bundle end time.